api/python/provisioner/commands/check.py

```diff
@@ -137,6 +137,37 @@ def is_set(self) -> bool:
         """
         return self._verdict is not None
 
+    @property
+    def is_critical(self) -> bool:
+        """
+        Return True if check result is marked as critical and False otherwise
+        """
+        return bool(self._critical)
```

Comment:
I guess that conversion is required since `None` is possible. But it means that `None` would be presented as non critical. Don't you think that we can simplify that and use `False` as default instead.

Reply:
Yes, I thought about that. The reason why I decided to keep `None` as one of the possible values is to mark that this property wasn't set by the user. This information is not used yet, but maybe it is good to store it and use it for unit tests in the future.
Also, as this property is optional for use, I treat it the same as a non-critical value if it would have been set to `False` value.

Reply:
ok, makes sense about `is_set` api then